packages/wordpress-kit/__tests__/GraphqlClientFactory.test.ts

```diff
@@ -28,3 +28,12 @@ test('request and response middleware add appropriate headers', async () => {
 	expect(res.data).toEqual(basicPostsQuery);
 	expect(res.headers.has('surrogate-key-raw')).toBeTruthy();
 });
+
+test('client is configured for GET requests', async () => {
+	const client = new GraphqlClientFactory('http://my-wp.test/graphql').create();
+	const getClient = new GraphqlClientFactory('http://my-wp.test/graphql', {
+		method: 'GET',
+	}).create();
+
+	expect(client).not.toMatchObject(getClient);
```

Comment:
We can be a bit more specific here:
```suggestion
	const getClient = new GraphqlClientFactory('http://my-wp.test/graphql', {
		method: 'GET',
	});

	expect(getClient.options.method).toEqual('GET');
```